src/main/java/org/opentripplanner/gtfs/mapping/LocationMapper.java

```diff
@@ -31,8 +32,8 @@ FlexStopLocation map(org.onebusaway.gtfs.model.Location orginal) {
   private FlexStopLocation doMap(org.onebusaway.gtfs.model.Location gtfsLocation) {
     FlexStopLocation otpLocation = new FlexStopLocation(mapAgencyAndId(gtfsLocation.getId()));
 
-    otpLocation.setName(gtfsLocation.getName());
-    otpLocation.setUrl(gtfsLocation.getUrl());
+    otpLocation.setName(new NonLocalizedString(gtfsLocation.getName()));
+    otpLocation.setUrl(new NonLocalizedString(gtfsLocation.getUrl()));
```

Comment:
Are these guaranteed to be non-null?

Reply:
According to GTFS spec at least no. Especially the URL seems to be causing issues as it was change to non null in the schema as well but it's optional in the gtfs spec.

Reply:
I was too quick to comment here, I thought this mapping was for regular GTFS stops but I think this is something else?

Reply:
Passing a null to the constructor of `NonLocalizedString` will result in an exception. I've made a convenience method to make dealing with this a little easier: https://github.com/opentripplanner/OpenTripPlanner/pull/3971

Reply:
This is fixed with https://github.com/opentripplanner/OpenTripPlanner/pull/3971